src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/ChainPal.cs

```diff
@@ -13,6 +13,11 @@ namespace Internal.Cryptography.Pal
 {
     internal sealed partial class ChainPal
     {
+        public static IChainPal FromHandle(IntPtr chainContext)
+        {
+            throw new PlatformNotSupportedException();
```

Comment:
This is not temporary. In OpenSsl there is a stack of certificates, and not a separate object to represent a chain. We could add add our own abstraction to enable IntPtr, but the main purpose of the IntPtr is for interop with the technology (CAPI, OpenSsl, etc)